.github/workflows/05-prepare-release.yml

```diff
@@ -41,6 +42,7 @@ jobs:
       - uses: actions/upload-artifact@v4
         with:
           name: context
+          include-hidden-files: true
```

Comment:
do we also have this on trunk?

Reply:
For reference: https://github.com/shopware/shopware/blob/trunk/.github/workflows/05-prepare-release.yml#L46